### PR TITLE
Edu 1995 school unit tests

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1319

They pass. Yay. If we need more tests, I can certainly fill them out, but this seemed to be in line with the amount of testing in the other schemaform unit tests.

### CODE REVIEW

File: test/edu-benefits/1995/config/schoolSelection-newSchool.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    ReactTestUtils.findRenderedComponentWithType(form, Form).onSubmit({
+      preventDefault: f => f
+    });
+
+    // Expect these fields to require validation
+    expect(formDOM.querySelector('.usa-input-error #root_newSchoolName')).to.not.be.null;
+    expect(formDOM.querySelector('.usa-input-error #root_educationType')).to.not.be.null;
+  });
+
+  it('should have no errors with only required info filled in', () => {
```
Comment (jbalboni) [inline test/edu-benefits/1995/config/schoolSelection-newSchool.unit.spec.jsx]: I think we should flip this around and test that we get the right number of required errors if you submit without filling anything in. This scenario should be covered by the e2e tests.

File: test/edu-benefits/1995/config/schoolSelection-newSchool.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+import Form from 'react-jsonschema-form';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1995/config/form';
+
+describe('Edu 1995 schoolSelection', () => {
```
Comment (jbalboni) [inline test/edu-benefits/1995/config/schoolSelection-newSchool.unit.spec.jsx]: Should also have a test that checks that the Address fields are shown when you pick the right education type.

**Review by jbalboni — COMMENTED** [review]
Looks good, added some test suggestions.

File: test/edu-benefits/1995/config/schoolSelection-newSchool.unit.spec.jsx (hunk 2 above)
Comment (cvalarida) [inline test/edu-benefits/1995/config/schoolSelection-newSchool.unit.spec.jsx]: I thought about that one, but isn't that going away? Or was what I'm thinking the bit where we conditionally add an option to the education type?

Comment (jbalboni) [inline test/edu-benefits/1995/config/schoolSelection-newSchool.unit.spec.jsx]: You're thinking of the additional option in the dropdown, which has already been removed. This is staying.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**